dds_cli/__main__.py

```diff
@@ -744,24 +523,25 @@ def common_options_project(f):
     help="The name of the Principal Investigator",
 )
 @click.option(
```

Comment:
click doesn't really specify in the cli that this option can be used multiple times, so should we put that detail in the description?

Reply:
Fixed!